flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/StreamOperatorStateHandler.java

```diff
@@ -174,6 +180,7 @@ public OperatorSnapshotFutures snapshotState(
         return snapshotInProgress;
     }
 
+    @SuppressWarnings("checkstyle:RegexpSingleline")
```

Comment:
What is this suppression for?

Reply:
I don't know how that ended up there... 🙈 removing